utils/unittest/fixtures.go

```diff
@@ -417,7 +417,7 @@ func BlockHeaderFixture(opts ...func(header *flow.Header)) *flow.Header {
 
 func CidFixture() cid.Cid {
 	data := make([]byte, 1024)
-	_, _ = rand.Read(data)
+	_, _ = crand.Read(data)
```

Comment:
it'd be best not to swallow this error. Maybe update the method either to accept a testing object and require no error, or just return the error. It doesn't look like this fixture is even used.

Reply:
I would rather delete non-used functions. Thanks for the flag!

Reply:
At the same time, for fixtures I think failing randomness would just lead to the byte array remaining all zeros, which is still a valid "fixture".